charts/astronomer/values.yaml

```diff
@@ -107,7 +107,10 @@ houston:
 
   # Any Houston configuration. Reference here:
   # https://github.com/astronomer/houston-api/blob/master/config/default.yaml
-  config: {}
+  config:
```

Comment:
@sjmiller609 we can revert this back. This section is reserved for end-user overrides. Anything we want to package and ship with the distribution can be added here: https://github.com/astronomer/astronomer/blob/master/charts/astronomer/templates/houston/houston-configmap.yaml#L54-L62, which already has 0.11.0.